### PR TITLE
Deletelast

### PR DESCRIPTION
**Pull request by ssesfahani**
image

### CODE REVIEW

File: cogs/deletelast/deletelast.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+log: logging.Logger = logging.getLogger(__name__)
+
+class DeleteLast(commands.Cog):
+    """ Purge the last N messages to repent sins. """
+
+    def __init__(self, bot: Zhenpai):
+        self.bot = bot
+
+    def cog_unload(self):
+        self.check_reminders.cancel()
```
Comment (ssesfahani) [inline cogs/deletelast/deletelast.py:16]: I don't actually know what this does tbh

**Review by hbkse — APPROVED** [review]
just merge this shit if it's done probably works with no problems right surely wont delete all chat history and ban everyone 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯 🚀 👍 💯

File: cogs/deletelast/deletelast.py (hunk 1 above)
Comment (hbkse) [reply cogs/deletelast/deletelast.py:16]: oh you dont need this it actually prob errors, did you copy this from another modules, iirc it's cus the reminders cog has/had a background task and when the cog unloads you want to kill the task

**Comment by ssesfahani** [comment]
Can't merge but you got it

**Pull request merged**